website/docs/r/stack_hci_deployment_setting.html.markdown

```diff
@@ -0,0 +1,585 @@
+
+---
+
+A `intent` block supports the following:
+
+* `name` - (Required) Specifies the name of the intent. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `adapter` - (Required) Specifies a list of ID of network interfaces used for the network intent. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `traffic_type` - (Required) Specifies a list of network traffic types. Possible values are `Compute`, `Storage`, `Management`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `adapter_property_override` - (Optional) A `adapter_property_override` block as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `qos_policy_override` - (Optional) A `qos_policy_override` block as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `virtual_switch_configuration_override` - (Optional) A `virtual_switch_configuration_override` block as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `override_adapter_property_enabled` - (Optional) This parameter should only be modified based on your OEM guidance. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `override_qos_policy_enabled` - (Optional) This parameter should only be modified based on your OEM guidance. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `override_virtual_switch_configuration_enabled` - (Optional) This parameter should only be modified based on your OEM guidance. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `ip_pool` block supports the following:
+
+* `ending_address` - (Required) Specifies starting IP address for the management network. A minimum of six free, contiguous IPv4 addresses (excluding your host IPs) are needed for infrastructure services such as clustering. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `starting_address` - (Required) Specifies ending IP address for the management network. A minimum of six free, contiguous IPv4 addresses (excluding your host IPs) are needed for infrastructure services such as clustering. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `observability` block supports the following:
+
+* `episodic_data_upload_enabled` - (Required) Whether to collect log data to facilitate quicker issue resolution. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
```

Comment:
seems should be named as `data_logging_enabled`?

Reply:
can we keep this same to the swagger for more consistency.